spec/index.bs

```diff
@@ -2179,12 +2211,83 @@ interface GPUTextureUsage {
+                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be less than or equal to
+                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be less than
+                                        or equal to |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be 1.
+                                    :: |descriptor|.{{GPUTextureDescriptor/format}} must not be a compressed format or depth/stencil format.
+                                </dl>
+                            - If |descriptor|.{{GPUTextureDescriptor/sampleCount}} > 1 (|descriptor|.{{GPUTextureDescriptor/sampleCount}} is 4):
+                                - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be 1.
+                                - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be 1.
+                                - |descriptor|.{{GPUTextureDescriptor/usage}} must not include the {{GPUTextureUsage/STORAGE}} bit.
+                                - |descriptor|.{{GPUTextureDescriptor/format}} must be a [=renderable format=].
+                            - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be less than or equal to
+                                [$maximum mipLevel count$](|descriptor|.{{GPUTextureDescriptor/dimension}}, |descriptor|.{{GPUTextureDescriptor/size}}).
+                            - |descriptor|.{{GPUTextureDescriptor/usage}} must be a combination of {{GPUTextureUsage}} values.
+                            - If |descriptor|.{{GPUTextureDescriptor/usage}} includes the {{GPUTextureUsage/RENDER_ATTACHMENT}} bit,
+                                |descriptor|.{{GPUTextureDescriptor/format}} must be a [=renderable format=].
+                            - If |descriptor|.{{GPUTextureDescriptor/usage}} includes the {{GPUTextureUsage/STORAGE}} bit,
+                                |descriptor|.{{GPUTextureDescriptor/format}} must be listed in [[#plain-color-formats]] table
+                                with {{GPUTextureUsage/STORAGE}} capability.
+                            - Some |descriptor|.{{GPUTextureDescriptor/format}}s require [=feature=]s when creating a {{GPUDevice}}:
+                                - {{GPUTextureFormat/"depth24unorm-stencil8"}} requires [[#depth24unorm-stencil8]].
+                                - {{GPUTextureFormat/"depth32float-stencil8"}} requires [[#depth32float-stencil8]].
+                                - BC compressed format requires [[#texture-compression-bc]].
+                            - If |descriptor|.{{GPUTextureDescriptor/format}} is a compressed format:
```

Comment:
nit: This "If" isn't necessary; texel block width/height are defined for all formats.

Reply:
Agreed. Done. And I moves them to the 'dimension' part above (it is not a `format` part any more).